funnel/views/project.py

```diff
@@ -276,37 +277,32 @@ class ProjectView(
     ProjectViewMixin, DraftViewMixin, UrlChangeCheck, UrlForView, ModelView
 ):
     @route('')
-    @render_with('project.html.jinja2')
+    @render_with(html_in_json('project.html.jinja2'))
     @requires_roles({'reader'})
     def view(self):
-        transition_form = ProjectTransitionForm(obj=self.obj)
-        rsvp_form = RsvpTransitionForm()
-        current_rsvp = self.obj.rsvp_for(current_auth.user)
-        featured_proposals = self.obj.proposals.filter_by(featured=True)
         return {
             'project': self.obj.current_access(),
-            'current_rsvp': current_rsvp,
-            'rsvp_form': rsvp_form,
-            'transition_form': transition_form,
-            'featured_proposals': featured_proposals,
+            'featured_proposals': [
+                _p.current_access()
+                for _p in self.obj.proposals.filter_by(featured=True)
+            ],
         }
 
     @route('sub')
     @route('proposals')
-    @render_with('project_submissions.html.jinja2')
+    @render_with(html_in_json('project_submissions.html.jinja2'))
     @requires_roles({'reader'})
     def view_proposals(self):
-        cfp_transition_form = ProjectCfpTransitionForm(obj=self.obj)
         return {
-            'project': self.obj,
-            'cfp_transition_form': cfp_transition_form,
+            'project': self.obj.current_access(),
+            'submissions': self.obj.proposals,
```

Comment:
Needs `current_access`. Also needs consistency between 'proposal' and 'submission'. We are still using the term 'proposal' in all internal references, but we should switch over as a planned move.